RadixWallet/Clients/ProfileStore/ProfileStore.swift

```diff
@@ -32,6 +33,11 @@
 /// async read `device.name` and `device.model` and update the Profile's header's `creatingDevice`
 /// and `lastUsedOnDevice` to use these values.
 ///
+/// If a user creates a Profile using Account Recovery Scan, app is suppose to call `finishOnboarding:with`,
```

Comment:
I can't believe I am the one to spot spelling mistakes :), but should be `supposeD` instead of `suppose`? And probably should add an THE in `next time user starts THE app`?